<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>140</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>This is the label</string>
    </property>
   </widget>
   <widget class="QPushButton" name="left_Button">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>60</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Left</string>
    </property>
   </widget>
   <widget class="QPushButton" name="show_Button">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Show</string>
    </property>
   </widget>
   <widget class="QPushButton" name="hide_Button">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>60</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Hide</string>
    </property>
   </widget>
   <widget class="QPushButton" name="right_Button">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>60</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Right</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="lock_CheckBox">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>520</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Lock/Unlock</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Test1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Test2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Test3</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QDialogButtonBox" name="buttonBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>320</y>
      <width>166</width>
      <height>22</height>
     </rect>
    </property>
    <property name="standardButtons">
     <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
